src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -490,6 +490,16 @@ public override Symbol VisitNamedType(NamedTypeSymbol sourceType)
                     var typeMap = new TypeMap(otherTypeParameters, otherTypeArguments, allowAlpha: true);
                     return typeMap.SubstituteNamedType(otherDef);
                 }
+                else if (sourceType.IsTupleType)
+                {
+                    var otherDef = (NamedTypeSymbol)this.Visit(originalDef.TupleUnderlyingType);
```

Comment:
>originalDef [](start = 63, length = 11)

Minor thing. Why are we operating of off definition. Even though it is the same thing for a tuple, it would be clearer to operate of off sourceType, I believe.